src/main/java/racingcar/view/dto/CarNames.java

```diff
@@ -0,0 +1,66 @@
+package racingcar.view.dto;
+
+import racingcar.util.ErrorMessage;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class CarNames {
+    private static final String VALID_STRING_REGEX = "(\\w)+";
```

Comment:
정규표현식에서 `\w`는 WORD를 의미하는데요. WORD가 더 어울리지 않을까요?
```suggestion
    private static final String VALID_WORD_REGEX = "(\\w)+";
```

Reply:
WORD가 더 어울리네욥...! 정규 표현식에는 String이라는 개념은 존재하지 않으니..